__tests__/app.test.js

```diff
@@ -48,25 +44,33 @@ describe('/api', () => {
 
     describe('GET /api/articles/:article_id', () => {
         test('200: Returns the desired article when given an article_id ', () => {
-            
             return request(app)
             .get('/api/articles/3')
             .expect(200)
             .then((response) => {
-                expect(response.body.article).toEqual(
-                {
-                    article_id: 3,
-                    title: "Eight pug gifs that remind me of mitch",
-                    topic: "mitch",
-                    author: "icellusedkars",
-                    body: "some gifs",
-                    created_at: '2020-11-03T09:12:00.000Z',
-                    votes: 0,
-                })
-
-            })
-            
+                expect(response.body.article).toEqual(expect.objectContaining(
+                    {
+                        article_id: 3,
+                        title: "Eight pug gifs that remind me of mitch",
+                        topic: "mitch",
+                        author: "icellusedkars",
+                        body: "some gifs",
+                        created_at: '2020-11-03T09:12:00.000Z',
+                        votes: 0,
+                    })
+                ) 
+            })  
         });
+        test('200: An article response object should now include the property "comment_count". ', () => {
+            return request(app)
+            .get('/api/articles/3')
+            .expect(200)
+            .then((response) => {
+                console.log(response.body.article)
+                expect(response.body.article).toHaveProperty('comment_count')})
+            })
+
+
```

Comment:
Could you test for an invalid id? /api/articles/bananas?

Reply:
I remember Emily telling me that it wasn't necessary to handle that error in this test as i've already handled the invalid id error in a previous test? The previous test uses the same endpoint: /api/articles/:articles_id.